src/cli/commandModules/copy.ts

```diff
@@ -9,6 +9,7 @@ import {
   CopyArgs,
   copy,
 } from "../..";
+import { DOCS } from "../../bluehawk/const";
```

Comment:
Do we do this anywhere else?

Reply:
Removed project + docs links as per slack discussion